build/composer-dependency-analyser.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types = 1);
+
+use ShipMonk\ComposerDependencyAnalyser\Config\Configuration;
+use ShipMonk\ComposerDependencyAnalyser\Config\ErrorType;
+
+$config = new Configuration();
+
+return $config
+	->addPathToScan(__DIR__ . '/../bin', true)
+	->ignoreErrors([ErrorType::UNUSED_DEPENDENCY])
+	->ignoreErrorsOnPackage('phpunit/phpunit', [ErrorType::DEV_DEPENDENCY_IN_PROD]) // prepared test tooling
```

Comment:
All violations are within `src/Testing`, would you like to ignore this error only there?

*(that might force people to place all PHPUnit-related stuff there)*

Reply:
Personally, I'd keep it as is. Resolving thread.